etc/logrotate.d/chia-monitor

```diff
@@ -0,0 +1,14 @@
+# Logrotate file for chia-monitor.service
+
+/var/log/chia-monitor.log {
+    compress
+    notifempty
+    daily
+    rotate 3
+    create 0600 root root
```

Comment:
This should be `create 640 syslog adm` so that non-root users can also read the rotated log files.